pkg/components/exporter/exporter.go

```diff
@@ -132,6 +137,19 @@ func NewDaemonSet(detail components.Detail, k *v1alpha1.KeplerInternal) *appsv1.
 	}
 }
 
+func MountRedfishSecretToDaemonSet(ds *appsv1.DaemonSet, secret *corev1.Secret) {
+	spec := ds.Spec.Template.Spec
+	spec.Containers[IdxKeplerContainer].Command = append(spec.Containers[IdxKeplerContainer].Command, REDFISH_ARGS)
+	spec.Containers[IdxKeplerContainer].VolumeMounts = append(spec.Containers[IdxKeplerContainer].VolumeMounts,
+		corev1.VolumeMount{Name: "redfish-cred", MountPath: "/etc/redfish", ReadOnly: true})
+	spec.Volumes = append(spec.Volumes,
+		k8s.VolumeFromSecret("redfish-cred", secret.ObjectMeta.Name))
+	ds.Spec.Template.Spec = spec
+	ds.Spec.Template.Annotations = map[string]string{
+		REDFISH_ANNOTATION: secret.ResourceVersion,
+	}
```

Comment:
I think it may be sufficient to just annotate the ds instead of the template. Lets handle this in a different PR.